README.md

```diff
@@ -14,6 +14,8 @@ to download model checkpoints.
 
```

Comment:
Btw @rajatsen91 and I have recently updated the repo and will add a bunch of stuff to this README. Let's get this PR in before that. What about
1. Let's keep the environment files
2. Introduce `pip install timesfm` as another (preferred) option separately at the start of the README.

Reply:
restored the environment files, removed the timesfm[cpu] option, just going with timesfm